static/sass/_layout-css.js

```diff
@@ -0,0 +1,18 @@
+import { css } from "lit";
+
+export const LAYOUT_CSS = css`.layout {
```

Comment:
This would be more clear if you added a newline right at the start of the tagged string.